tasks/ollama-rag-chatbot-fn/task.yaml

```diff
@@ -0,0 +1,13 @@
+id: ollama-rag-endpoint
+name: "RAG endpoint"
+description: Ceates an endpoint for a RAG chatbot from a dataset containing the index generated by the RAG-index template
```

Comment:
This is not specifically RAG endpoint, it can be a standard LLM as well so lets update the id, name, and description accordingly.